Source/ProgressionSystemRuntime/Public/Components/PSHUDComponent.h

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Data/PSTypes.h"
+#include "Data/PSDataAsset.h"
```

Comment:
You don't need this include at all, any dependencies to any pointer (expect Parent class) can be replaced by 'class' in property\function name or as class forward above the class

Reply:
I've tried this and gives me errors. 
![image](https://github.com/h4rdmol/ProgressionSystem/assets/5227233/2f72e4cb-1ac5-43ff-988e-18e4c24ebffa)
![image](https://github.com/h4rdmol/ProgressionSystem/assets/5227233/dcb0cadc-3aba-4aad-a7b1-26368abf8caf)

Reply:
There should be only one GetProgressionSystemDataAsset(), in Subsystem, but not definitely in HUD. Instead, HUD code should use `UPSDataAsset::Get().GetSomething()` directly

Reply:
done